### ISSUE TITLE
Exception raised when sending a PENALTY to player number 4

### ISSUE THREAD

**Issue by omichel** [RoboCup-Humanoid-TC/GameController#127]
When sending a penalty command to player 4 of team 1, I am getting the following error in the GameController:
```
Exception in thread "Thread-101" java.lang.ArrayIndexOutOfBoundsException: 4
	at data.states.AdvancedData.getRemainingPenaltyTime(Unknown Source)
	at data.states.AdvancedData.updateTimes(Unknown Source)
	at controller.net.Sender.run(Unknown Source)
```
It is [documented](https://github.com/RoboCup-Humanoid-TC/GameController/blob/master/AutoRefAPI.md#penalty) that player_id should be in the range 1 to 4.

**Comment by medrimonia** [comment]
It sounds like something that was raised in #123 and solved with PR #125, have you pulled and rebuild the GameController since?

**Comment by omichel** [comment]
Yes, after pulling and recompiling the issue seems to be resolved. Thanks!
